.projenrc.js

```diff
@@ -56,4 +63,7 @@ project.deps.removeDependency("@aws-cdk/core", DependencyType.RUNTIME);
 project.deps.removeDependency("@aws-cdk/assert");
 project.deps.removeDependency("@aws-cdk/assertions");
 project.deps.removeDependency("monocdk", DependencyType.RUNTIME);
-project.synth();
\ No newline at end of file
+project.deps.removeDependency("constructs", DependencyType.RUNTIME);
+project.deps.addDependency("monocdk@1.123.0", DependencyType.DEVENV);
```

Comment:
Can you extract the version out as a constant?

Reply:
done